ci-operator/step-registry/ipi/conf/vsphere/zones/multisubnets/ipi-conf-vsphere-zones-multisubnets-commands.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# ensure LEASED_RESOURCE is set
+if [[ -z "${LEASED_RESOURCE}" ]]; then
+  echo "Failed to acquire lease"
+  exit 1
+fi
+
+echo "$(date -u --rfc-3339=seconds) - sourcing context from vsphere_context.sh..."
+# shellcheck source=/dev/null
+declare vsphere_datacenter
+declare vsphere_url
+source "${SHARED_DIR}/vsphere_context.sh"
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+base_domain=$(<"${SHARED_DIR}"/basedomain.txt)
+machine_cidr=$(<"${SHARED_DIR}"/machinecidr.txt)
+
+
+cat >> "${CONFIG}" << EOF
+baseDomain: $base_domain
+controlPlane:
+  name: "master"
+  replicas: 3
+  platform:
+    vsphere:
+      zones:
+       - "us-east-1"
+       - "us-east-2"
+       - "us-east-3"
+compute:
+- name: "worker"
+  replicas: 3
+  platform:
+    vsphere:
+      zones:
+       - "us-west-1"
+platform:
+  vsphere:
+    apiVIP:
+    ingressVIP:
+    vCenter: "${vsphere_url}"
+    username: "${GOVC_USERNAME}"
+    password: ${GOVC_PASSWORD}
+    network: ${LEASED_RESOURCE}
+    datacenter: "${vsphere_datacenter}"
+    cluster: vcs-mdcnc-workload-1
+    defaultDatastore: mdcnc-ds-shared
+    failureDomains:
+    - name: us-east-1
+      region: us-east
+      zone: us-east-1a
+      topology:
+        computeCluster: /${vsphere_datacenter}/host/vcs-mdcnc-workload-1
+        networks:
+        - ${LEASED_RESOURCE}
+        datastore: mdcnc-ds-shared
+    - name: us-east-2
+      region: us-east
+      zone: us-east-2a
+      topology:
+        computeCluster: /${vsphere_datacenter}/host/vcs-mdcnc-workload-2
+        networks:
+        - ocp-ci-seg-20
```

Comment:
This setting is for the `multipsubnets` meaning, right? Once we use it as hard code, we should never add it to prow cluster profile any more, right?

Reply:
sorry, I'm not very clear about your comments.  do you mean multisubnets maybe also need by workers, so i should not only hard code for master?

Reply:
In a common case, we should place all the nodes in `${LEASED_RESOURCE}`, this case is for `multsubnets`, so we set a hard-code segment for the domains on purpose, right?

Reply:
yes, that is.

Reply:
The `ocp-ci-seg-20` and `ocp-ci-seg-21` is newly created by DPP team, dedicated for this case?

Reply:
we applied for new subnets, but ran in to issue and they suggested to use these two segments.